tests/integration/security/external_ca/main_test.go

```diff
@@ -135,7 +146,6 @@ components:
 {{- else }}
```

Comment:
Same question as above, why are lines 139-144 needed? They don't seem to override anything. Can't you remove them and change line 146 to:
```
{{- if not .isConfigCluster}}
```